br/pkg/lightning/mydump/csv_parser.go

```diff
@@ -576,6 +593,21 @@ func (parser *CSVParser) readQuotedField() error {
 			if doubledDelimiter {
 				// consume the double quotation mark and continue
 				parser.recordBuffer = append(parser.recordBuffer, parser.quote...)
+			} else if parser.unescapedQuote {
```

Comment:
Should `"a""b"`  be parsed to `a""b` when `unescapedQuote` is `true`?

Reply:
No, the doc has an example

```
"The ""BIG"" boss"  -> The "BIG" boss
```

https://dev.mysql.com/doc/refman/8.0/en/load-data.html#load-data-file-location

Reply:
The rule is kind of strange.